tests/installation/releasenotes.pm

```diff
@@ -41,10 +41,14 @@ sub run() {
     if (check_var('VIDEOMODE', 'text')) {
         send_key 'tab';          # select tab area
     }
+
+    # no release-notes for WE and all modules
+    my @no_relnotes = qw(we lgm asmm certm contm pcm tcm wsm);
+    # no relnotes for ltss in QAM_MINIMAL
+    push @no_relnotes, qw(ltss) if get_var('QAM_MINIMAL');
     if (@addons) {
         for my $a (@addons) {
-            # no release-notes for WE and all modules
-            next if grep { $a eq $_ } qw(we lgm asmm certm contm pcm tcm wsm);
+            next if grep { $a eq $_ } (@no_relnotes);
```

Comment:
you don't need to wrap the array in an array. Perl doesn't know multi-dimensional arrays anyway. Just @no_relnotes will do